### ISSUE TITLE
"Number of subscriptions"

### ISSUE THREAD

**Issue by drmclean** [drmclean/monthlies#14]
I think this could be better. 

An unlabelled counter with some explanatory text above it is kinda guaranteed to go wrong I think. 

Suggestions. 

1. Label the counter:     Number of Subscriptions - <counter box>
2. Move the explanatory text either right above or right below the counter.

This could be an entire separate page so as not to confuse people?

**Comment by SarahHewett** [comment]
Yeah, I'd rather just get rid of the counter, but can't find how to do this. I'll look into labelling the counter and will move the tex to the bottom if I can't do that.
